### PR TITLE
fix(components): [message] set appentTo value

### LINKED ISSUE #8691: [Component] [message]ElementPlus组件从head头部引入时ElMessage组件使用异常，版本号2.7+
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.37`
- Element Plus Version: `2.2.7`
- Browser / OS: `Chrome/103.0.5060.66（正式版本）`
- Build Tool: `Webpack`

### Related Component

- `el-message`

### Reproduction Link

[JSFiddle](https://jsfiddle.net/wangnamin/gojwk1x6/2/)

### Steps to reproduce

将代码复制到html文件中执行，点击按钮，查看控制台

### What is Expected?

el-message组件正常弹出

### What is actually happening?

el-message组件没有正常弹出，在控制台有错误

### Additional comments

jsfiddle中的代码运行正常，需要将代码复制到html文件中执行，即可复现

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

fix #8691

### CODE REVIEW

File: packages/components/message/src/method.ts [hunk 1]
```diff
@@ -41,6 +41,9 @@ const normalizeOptions = (params?: MessageParams) => {
     ...options,
   }
 
+  if (!normalized.appendTo) {
+    normalized.appendTo = document.body
+  }
   if (isString(normalized.appendTo)) {
```
Comment (chenxch) [inline packages/components/message/src/method.ts:47]: Maybe use `else if` link,one judgment can be reduced.

Comment (btea) [reply packages/components/message/src/method.ts]: You are right.

**Review by chenxch — APPROVED** [review]
LGTM.

**Review by buqiyuan — APPROVED** [review]

File: packages/components/message/src/message.ts [hunk 2]
```diff
@@ -32,7 +31,6 @@ export const messageDefaults = mutable({
   zIndex: 0,
   grouping: false,
   repeatNum: 1,
-  appendTo: isClient ? document.body : (undefined as never),
```
Comment (tolking) [inline packages/components/message/src/message.ts:35]: It should not be removed here, it will affect the generation of the types

**Review by tolking — APPROVED** [review]

**Pull request merged**